jboss-eap-clustered-rhel-VMSS/README.md

```diff
@@ -37,31 +37,31 @@ Following is the Architecture :
 To learn more about JBoss Enterprise Application Platform, check out:
 https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.2/
 
-## Licenses and Costs 
+## Licenses, Subscriptions and Costs 
 
-This uses RHEL 8.0 image which is a PAY AS YOU GO image and doesn't require the user to license it, it will be licensed automatically after the instance is launched first time and user will be charged hourly in addition to Microsoft's Linux VM rates.  Click [here](https://azure.microsoft.com/en-gb/pricing/details/virtual-machines/linux/#red-hat) for pricing details.
+This template uses a RHEL 8.0 image which is a Pay-as-you-go (PAYG) image and does not require the user to license.  The VM will be licensed automatically after the instance is launched first time and the user will be charged hourly in addition to Microsoft's Linux VM rates.  Click [here](https://azure.microsoft.com/en-gb/pricing/details/virtual-machines/linux/#red-hat) for pricing details.
 
 ## Prerequisites
 
-1. Azure Subscription with specified payment method (RHEL 8.0 is a market place product and requires payment method to be specified in Azure Subscription)
+1. Azure Subscription with specified payment method (RHEL 8.0 is a marketplace product and requires a payment method to be specified in Azure Subscription)
 
-2. To deploy the template, you will need to:
+2. To deploy the template, you will need:
 
-    - Choose an admin username and password for your VM.  
+    -  
 
-    - Choose a EAP username and password to enable the EAP manager UI and deployment method.
+    - *EAP Username** and password to enable the EAP manager UI and Deployment Method.
```

Comment:
Modified this line as follows as per the feedback from other PRs for consistency. Please suggest

JBoss EAP Username and password